src/registerAdapters.ts

```diff
@@ -74,7 +76,23 @@ function registerAdapterForServer(serverName: string, server: Server, app: Glee,
       }
 
     }
-  } else {
+  }else if (['http', 'https'].includes(protocol)) {
+    if(remoteServers){
```

Comment:
There may be remote servers but none of them using the HTTP protocol. You have to check for the protocol here too.